test/setup.js

```diff
@@ -265,3 +268,23 @@ function withVersions (plugin, moduleName, range, cb) {
       describe(`with ${moduleName} ${v.range} (${v.version})`, () => cb(v.test))
     })
 }
+
+function createOperation (service) {
+  const timeouts = retry.timeouts(retryOptions)
+  return new RetryOperation(timeouts, Object.assign({ service }, retryOptions))
+}
+
+function retryOperation (operation, err) {
+  const shouldRetry = operation.retry(err)
+  if (shouldRetry) {
+    logAttempt(operation._options.service, operation._attempts, err.message)
+  }
+  return shouldRetry
+}
+
+function logAttempt (service, attempts, message) {
+  if (attempts > 2) {
```

Comment:
Locally it should generally not fail, unless you've just updated your containers. The long retry timeout is mostly because on CI sometimes it can take over 1-2 minutes and even more to start some of the services like Elasticsearch.

Reply:
What do you want to do, lower it to 2 or not defer at all?

Reply:
I'd not defer, this way feedback would be provided within 5 seconds locally, and on CI it only doesn't really make a difference.

Reply:
Okeydokey 👍